dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/AppConfiguration.java

```diff
@@ -43,6 +50,9 @@
     public static final String PATH_PATTERN = "/**";
     public static final String LOCALE_LANGUAGE_COOKIE = "language";
 
+    @Value("${max.global.qps.rate:100}")
+    private int globalLimitQps;
```

Comment:
The default value is best not to set a limit. Otherwise, when the user upgrades, ignoring this parameter may cause unnecessary trouble.

Reply:
You are right